### ISSUE TITLE
Script file is deleted upon deploy

### ISSUE THREAD

**Issue by pastadin** [hyperledger/composer#1096]
1. Go to the Web Playground: http://composer-playground.mybluemix.net/editor
2. Create an empty project (authenticate with github first)
3. Add a new model file:
```
/**
 * New model file
 */

namespace org.acme.model

enum Sex {
o MALE
   	o FEMALE
}


concept Address {
	o String house
    	o String street
    	o String county
    	o String postcode
    	o String country
}

participant Collector identified by email {
  o String email
  o String firstname
  o String surname
  o Address address
  o Sex sex
}


asset Marble identified by Id { 
	o String Id
      --> Collector owner
      o Integer diameter
      o String colour
}

transaction ChangeOwner identified by txId {
	o String txId
    	--> Marble marble
    	--> Collector newOwner
}
```
4. Deploy the network
5. Go back to Define and create a new script file.
6. Deploy the network
7. The script file should now be gone

Chrome 58
macOS 10.11.6

**Comment by dselman** [comment]
I think @nklincoln has also reproduced this.

**Comment by nklincoln** [comment]
closing as is a duplicate of #1093, fixed with PR #1094
